### PR TITLE
Cartpage

### PR DESCRIPTION
**Pull request by dev-prabhat**
Check only JS File -
Cart.js
CartCard.js
CartContext.js

Ignore All CSS files

### CODE REVIEW

File: src/Context/Cart-Context.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                if (index === -1) {
+                    return {
+                        ...state, cartItems: [...cartItems, { _id, name, image, qty: 1 }]
+                    }
+                }
+                return {
+                    ...state, cartItems: cartItems.map(item =>
+                        item._id === _id ? { ...item, qty: item.qty + 1 } : item
+                    )
+                }
```
Comment (zoef-yt) [inline src/Context/Cart-Context.js]: Suggestion can be done with the ternary operator

**Review by zoef-yt — APPROVED** [review]
The code looks. good work. good to merge.

File: src/pages/Product/Product.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+const Product = () => {
+    return (
+        <main className="productpage-grid-container">
+            <Sidebar />
+            <section className="content d-flex">
+                {
+                    ProductData.map((Obj) => (
+                        <ProductCard key={Obj._id} product={Obj} />
```
Comment (srishtigautam21) [inline src/pages/Product/Product.js]: Just a suggestion you could give more specific variable name like productObj

**Review by srishtigautam21 — APPROVED** [review]
The code is well understandable
good to merge

**Review by iRohitGaur — APPROVED** [review]
Work on the suggestions already given. Rest all is good. Nice work! Good to merge 👍🏻

**Pull request merged**